communities/src/lib.rs

```diff
@@ -114,6 +151,60 @@ decl_module! {
             info!(target: LOG, "registered community with cid: {:?}", cid);
         }
 
+        #[weight = 10_000]
+        pub fn add_location(origin, cid: CommunityIdentifier, location: Location) {
```

Comment:
I believe we should support adding several locations with one extrinsic. But we'll have to enforce a maximum number

Reply:
Opened issue https://github.com/encointer/pallets/issues/53 for this